tests/python/shared/fixtures/init.py

```diff
@@ -22,6 +22,19 @@
 CVAT_DB_DIR = ASSETS_DIR / "cvat_db"
 PREFIX = "test"
 
+DB_CONTAINER = "cvat_db"
+SERVER_CONTAINER = "cvat_server"
+ANNOTATION_CONTAINER = "cvat_worker_annotation"
+IMPORT_CONTAINER = "cvat_worker_import"
+EXPORT_CONTAINER = "cvat_worker_export"
```

Comment:
The widespread practice is to define such constants as Enum

Reply:
Updated